conductr_cli/conduct_info_list.py

```diff
@@ -38,7 +38,7 @@ def display_bundles_default(args, is_license_success, conductr_license, bundles)
     data.insert(0, {
         'id': 'ID',
         'name': 'NAME',
-        'compatibility_version': 'VER',
+        'tag': 'TAG',
```

Comment:
Not a bad idea. If tags is missing for one bundle it'll be missing for all. We could decide between the existing behaviour and tags based on this, and not have the fallback logic as a consequence.